components/alert/alert.css

```diff
@@ -0,0 +1,147 @@
+@import url('/css/allStyles.css');
+
+body {
+  background-image: none;
+}
+
+.wrapper {
+  width: 100%;
+  min-width: 100vh;
+  height: max-content;
+  margin: auto;
+}
+
+.wrapper .bottom-nav {
+  width: 100%;
```

Comment:
I think for the styles of common components like navbar, you should put them in the common stylesheet. ELse, you will have to add this code to every component's CSS file.